README.md

```diff
@@ -78,6 +78,7 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 
 ### General Web Development
 
+- [Codrops](https://tympanus.net/codrops/). Weekly news about qualitative news projects and libraries in Web Development and design
```

Comment:
would you mind adding a dot at the end ?

Reply:
IMHO, description should be improve a bit.

> Weekly news about qualitative news

there is a repetition of `news`, doesn't sound nice.

Reply:
will patch myself to close this.